docs/translating/polish.md

```diff
@@ -190,21 +190,21 @@ You can pin streams in the left-side panel.
 
 > Możesz przypiąć wybrane kanały w menu kontekstowym po lewej.
 
-**private message**: prywatna wiadomość
+**direct message**: Wiadomość bezpośrednia
 
 example:
 
-You can send a private message to other users in your organization.
+You can send a direct message to other users in your organization.
 
 > Możesz wysłać prywatną wiadomość do innych użytkowników w Twojej organizacji.
 
-**PM**: PM, translation could be confusing
+**DM**: DM, translation could be confusing
 
 example:
 
-Private messages are often abbreviated to PM.
+Direct messages are often abbreviated to DM.
 
-> Prywatne wiadomości są też nazywane PMami, od angielskiego "private message".
+> Prywatne wiadomości są też nazywane DMami, od angielskiego "direct message".
```

Comment:
@mateuszmandera can you help verify the polish translation guide changes for DMs?